### DISCUSSION TITLE
Adding scripted constraints to the GUI

### DISCUSSION THREAD

**Discussion by benayaspenasm** [nasa/EMTG#7] [Ideas]
I think the GUI should include all the options available in the emtgopt files. Otherwise, it could be misleading to users. One good example to improve is, specifically, the scripted constraints.

**Comment by ednasa** [comment]
Agreed. 
From what I have seen the scripted constraints functionality is very powerful. 
Unfortunately the creation of a GUI for this functionality can be pretty complicated and with limited staff/resources it typically goes to the backlog. It's definitely something that shouldn't be forgotten.

**Reply by am-nasa-gsfc** [reply]
Good note, we use scripted constraints in almost every mission at this point. Incorporating them in the GUI is something that's been on our radar for awhile now. We have some other GUI work upcoming so we'll be taking a look at including them then.
